tests/load_optimization_test.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+
+from floris import (
+    FlorisModel,
+    TimeSeries,
+    WindRose,
+)
+from floris.optimization.load_optimization.load_optimization import (
+    compute_farm_revenue,
+    compute_farm_voc,
+    compute_load_ti,
+    compute_net_revenue,
+    compute_turbine_voc,
+    find_A_to_satisfy_rev_voc_ratio,
+    get_max_powers,
+    get_rotor_diameters,
+    optimize_derate,
```

Comment:
There are not currently any tests for this---perhaps we could add a simple regression test?